addons/mail/static/src/core/common/core.scss

```diff
@@ -50,17 +50,56 @@
     z-index: 1;
 }
 
+// see `@mixin button-variant`, this is the implementation without requiring `.btn` classname
+@mixin o-mention-variant(
+    $background,
+    $border,
+    $color: color-contrast($background),
+    $hover-background: if($color == $color-contrast-light, shade-color($background, $btn-hover-bg-shade-amount), tint-color($background, $btn-hover-bg-tint-amount)),
+    $hover-border: if($color == $color-contrast-light, shade-color($border, $btn-hover-border-shade-amount), tint-color($border, $btn-hover-border-tint-amount)),
+    $hover-color: color-contrast($hover-background),
+    $active-background: if($color == $color-contrast-light, shade-color($background, $btn-active-bg-shade-amount), tint-color($background, $btn-active-bg-tint-amount)),
+    $active-border: if($color == $color-contrast-light, shade-color($border, $btn-active-border-shade-amount), tint-color($border, $btn-active-border-tint-amount)),
+    $active-color: color-contrast($active-background),
+    $disabled-background: $background,
+    $disabled-border: $border,
+    $disabled-color: color-contrast($disabled-background)
+) {
+    color: #{$color};
+    background-color: #{$background};
+    border-color: #{$border};
+    &:hover {
+        color: #{$hover-color};
+        background-color: #{$hover-background};
+        border-color: #{$hover-border};
+    }
+    &:focus {
+        box-shadow: #{to-rgb(mix($color, $border, 15%))};
+    }
+    &:active {
+        color: #{$active-color};
+        background-color: #{$active-background};
+        border-color: #{$active-border};
+    }
+    &:disabled {
+        color: #{$disabled-color};
+        background-color: #{$disabled-background};
+        border-color: #{$disabled-border};
+    }
+}
+
 a.o_mail_redirect, a.o_channel_redirect, a.o-discuss-mention {
-    @include button-size($btn-padding-y-sm, $btn-padding-x-sm, $btn-font-size-sm, $btn-border-radius-sm);
-    padding: map-get($spacers, 1);
+    @include rfs($btn-font-size-sm, $btn-font-size-sm);
```

Comment:
Just found something funky with this line : it actually renders as `0.8125rem: 0.8125rem;`
Which is definitively not what's expected :sweat_smile:

The reason is that the second parameter is the name of the property to which you want to apply the calculated size.

Do we still need it (applied to `font-size`) or can we get rid of this (currently) invalid, and thus unapplied, rule ?
@zel-odoo @alexkuhn

Reply:
Good catch! Personally I don't complaint with the current sizing of the mentions, so let's remove it!

Reply:
@alexkuhn Here is the fix :wink: https://github.com/odoo/odoo/pull/201517